### PR TITLE
feat(fin360): Get all Transactions and Account details feature added

### PR DESCRIPTION
**Pull request by kmittal01**

### CODE REVIEW

File: fin360/fin360.py [hunk 1]
```diff
@@ -44,3 +44,16 @@ def authenticate(self, email, password):
         self.access_token = response.json()['access_token']
 
         return AttrDict(response.json())
+
+    def get_transactions_with_details(self, accountid):
+
+        url = "https://www.fin360.in/bank-account/api/v1/transactionsWithDetails/" + accountid + ".json"
```
Comment (kgritesh) [inline fin360/fin360.py]: please add a documentation of the response format ie what is returned by the api. An eg transaction would be great

Comment (kgritesh) [inline fin360/fin360.py]: as i mentioned earlier, urls must be constructed from a base_url which is constant. Also use string formatting 
  
  `"https://www.fin360.in/bank-account/api/v1/transactionsWithDetails/{}.json".format(account_id)`

File: fin360/fin360.py [hunk 2]
```diff
@@ -44,3 +44,16 @@ def authenticate(self, email, password):
         self.access_token = response.json()['access_token']
 
         return AttrDict(response.json())
+
+    def get_transactions_with_details(self, accountid):
```
Comment (kgritesh) [inline fin360/fin360.py]: change param to account_id. Also need to check if access_token is set or not.

**Comment by kmittal01** [comment]
review changes done.

**Pull request merged**